bayesian/utils.py

```diff
@@ -381,10 +381,17 @@ def get_analyses_for_snyk(self):
 
             if "cve" in result_data[0]:
                 # if cve if present
+                rec_version = result_data[0].get('package', {}).get(
```

Comment:
Can you rename variable according to db field name? It should be `latest_non_cve_version` in this case.

Reply:
its recommended version.